content/blog/tidymodels-xgboost/index.Rmd

```diff
@@ -20,6 +20,19 @@ The [xgboost](https://xgboost.readthedocs.io/en/stable/r_docs/R-package/docs/ind
 
 In addition to all the new features and improvements that are now available for users relying on CRAN versions of packages, there are also a few breaking changes. Specifically between version 1.x and 2.x of the xgboost library. The xgboost team has kindly provided a [migration guide](https://xgboost.readthedocs.io/en/stable/R-package/migration_guide.html) for how to update your code if you are upgrading from before version 2.x.
 
-If you are using xgboost purely through tidymodels via functions like `parsnip::boost_tree()` and `embed::step_discretize_xgb()`, you should not need to change anything, as we have updated our packages to work with both the new and old versions of xgboost. If you are having any issues, please let us know by filing an issue for the affected package.
+If you are using xgboost purely through tidymodels via functions like [`parsnip::boost_tree()`](https://parsnip.tidymodels.org/reference/boost_tree.html) and [`embed::step_discretize_xgb()`](https://embed.tidymodels.org/reference/step_discretize_xgb.html), you should not need to change anything, as we have updated our packages to work with both the new and old versions of xgboost. If you are having any issues, please let us know by filing an issue for the affected package.
+
+We look forward to integrating parsnip more deeply into these new changes, such as support for [categorical predictors](https://xgboost.readthedocs.io/en/stable/tutorials/categorical.html) and [quantile regression](https://xgboost.readthedocs.io/en/stable/python/examples/quantile_regression.html#quantile-regression).
+
+Here are the package that we've updated or helped the maintainers update 
+
+- [parsnip](https://parsnip.tidymodels.org/news/index.html#parsnip-140)
+- [tidypredict](https://tidypredict.tidymodels.org/news/index.html#tidypredict-100)
+- [embed](https://embed.tidymodels.org/news/index.html#embed-121)
+- [butcher](https://butcher.tidymodels.org/news/index.html#butcher-040)
+- [lime](https://github.com/tidymodels/lime)
+- [bundle](https://rstudio.github.io/bundle/dev/news/index.html#bundle-development-version)
+- [vetiver](https://rstudio.github.io/vetiver-r/dev/index.html)
```

Comment:
bundle is already released: https://rstudio.github.io/bundle/news/index.html#bundle-013

vetiver is submitted as of this morning but I haven't quite heard back yet

Reply:
vetiver is now released as well: https://rstudio.github.io/vetiver-r/news/index.html#vetiver-027